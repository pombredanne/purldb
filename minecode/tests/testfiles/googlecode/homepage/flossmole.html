

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"token": null, "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "projectHomeUrl": "/p/flossmole", "profileUrl": null, "domainName": null, "projectName": "flossmole", "loggedInUserEmail": null, "relativeBaseUrl": "", "assetHostPath": "http://www.gstatic.com/codesite/ph"};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 flossmole -
 
 
 Data collection and analysis about free, libre and open source software - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fflossmole%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fflossmole%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/flossmole">
 <a href="/p/flossmole/">
 
 
 <img src="/p/flossmole/logo?cct=1347303634"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/flossmole/"><span itemprop="name">flossmole</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/flossmole/"><span itemprop="description">Data collection and analysis about free, libre and open source software</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/flossmole/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/flossmole/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/flossmole/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/flossmole/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/flossmole/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/flossmole/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/flossmole/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/flossmole/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://dev.perl.org/licenses/" rel="nofollow">Artistic License/GPL</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:data">data</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:collection">collection</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:metrics">metrics</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:floss">floss</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:source">source</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:free">free</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:statistics">statistics</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:datamining">datamining</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:open">open</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:foss">foss</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/megansquire/">megansquire</a
>, 
 
 

 <a class="userlink"
 href="/u/112693733048749814994/">jhowi...@syr.edu</a
>, 
 
 

 <a class="userlink"
 href="/u/104060006849124834620/">crows...@syr.edu</a
>, 
 
 

 <a class="userlink"
 href="/u/118014214667565137938/">akwigg...@gmail.com</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">2 committers</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 <li class="psmeta"><b>Blogs</b></li>
 
 <li class="psmeta">
 <a href="http://flossmole.org" rel="nofollow">FLOSSmole Blog</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="http://flossmole.org" rel="nofollow">FLOSSmole Web Site</a>
 </li>
 
 <li class="psmeta">
 <a href="http://flossdata.syr.edu/data" rel="nofollow">All flat data files</a>
 </li>
 
 <li class="psmeta">
 <a href="http://flossmole.org/content/direct-db-access-flossmole-collection-available" rel="nofollow">Direct DB Access (sql)</a>
 </li>
 
 <li class="psgap"/>
 
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p>UPDATE: our web site is at <a href="http://flossmole.org" rel="nofollow">flossmole.org</a> </p><p>WE ARE NO LONGER ALLOWED TO HOST NEW PROJECT FILES HERE so please <a href="http://flossdata.syr.edu/data/" rel="nofollow">go to our web site for all files</a>. </p><p>FLOSSmole aims to: <ul><li>-provide raw data about free and open source projects </li><li>-provide summary reports about FLOSS projects and forges </li><li>-integrate donated data from other research teams </li><li>-provide some tools so that you can gather your own data </li></ul></p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

