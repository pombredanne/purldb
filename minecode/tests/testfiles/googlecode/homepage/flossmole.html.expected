{'content_license': None,
 'content_license_url': None,
 'description': u'Data collection and analysis about free, libre and open source software',
 'downloads_url': u'http://code.google.com/p/flossmole/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/flossmole/feeds',
 'home_url': u'http://code.google.com/p/flossmole/',
 'id': u'flossmole',
 'issues_url': u'http://code.google.com/p/flossmole/issues/list',
 'keywords': [],
 'labels': [u'collection',
            u'data',
            u'datamining',
            u'floss',
            u'foss',
            u'free',
            u'metrics',
            u'open',
            u'source',
            u'statistics'],
 'license': u'Artistic License/GPL',
 'license_url': u'http://dev.perl.org/licenses/',
 'name': u'flossmole',
 'peoples_url': u'http://code.google.com/p/flossmole/people/list',
 'vcs_url': u'http://code.google.com/p/flossmole/source/checkout',
 'wiki_url': u'http://code.google.com/p/flossmole/w/list'}